Not possible to import external stylesheets

### Is this a bug report?
No.

Currently, there is no way, as far as I know, to add an external stylesheet to the app. Notably, importing stylesheets in JS will not work because of CSS Modules. This makes it impossible to use certain 3rd party components.

One solution is to add a separate config for CSS/SCSS files imported from `node_modules`. I made a proof of concept commit [here](https://github.com/papermana/create-react-app/commit/cb035db73f3fbd2c638d50dd52af5f9f752c3021) that seems to work when I make those changes locally. I'm not sure if I'm not omitting anything in the config, though. Also, I haven't made any changes to the production config. I can make a proper PR if the changes here look good to others.

Should it work out of the box? With the following code
```
:global {
  @import '~normalize.css';
  @import '~bootstrap/dist/css/bootstrap.css';
}
```
`normalize.css` is included, but bootstrap's import results in 
```
./src/global-styles.scss
Module not found: Can't resolve '~bootstrap/dist/css/bootstrap.css' in '/Users/wiktor/Workspace/spideo-front/src'
```

--- a/README.md
+++ b/README.md
@@ -59,7 +59,7 @@ With this approach we can benefit from what is best from those two worlds. We ca
 How to use it? Since there is no global scope we need to implicitly import our variables/mixins etc. Check the `src/Components/App/style.scss` file. On the first line it imports the shared styles with variables and mixins.
 
 #### 4. How do I import global styles from third-party packages?
-Importing any stylesheet from JS (with CSS Modules enabled) will convert class names into unique identifiers. By design, this prevents them from being used for global styles. If you have, say, a third-party component that provides some "regular" SCSS/CSS code, you will want to, instead, import it from an SCSS file like so:
+Importing any stylesheet from JS (with CSS Modules enabled) will convert class names into unique identifiers. By design, this prevents them from being used for global styles. If you have, say, a third-party component that provides some "regular" SCSS/CSS code, you will want to import it from an SCSS file instead, like so:
 ```
 :global {
   @import '~react-select/dist/react-select';

--- a/README.md
+++ b/README.md
@@ -57,3 +57,11 @@ to:
 With this approach we can benefit from what is best from those two worlds. We can use SCSS syntax for mixins, variables etc. and still have our rules scoped to the component ;)
 
 How to use it? Since there is no global scope we need to implicitly import our variables/mixins etc. Check the `src/Components/App/style.scss` file. On the first line it imports the shared styles with variables and mixins.
+
+#### 4. How do I import global styles from third-party packages?
+Importing any stylesheet from JS (with CSS Modules enabled) will convert class names into unique identifiers. By design, this prevents them from being used for global styles. If you have, say, a third-party component that provides some "regular" SCSS/CSS code, you will want to, instead, import it from an SCSS file like so:
+```
+:global {
+  @import '~react-select/dist/react-select';
+}
+```

